CHANGELOG.md

```diff
@@ -25,7 +25,11 @@
 * **Bolt issues a warning when inventory overrides a CLI option** ([#1341](https://github.com/puppetlabs/bolt/issues/1341))
 
   Bolt issues a warning when an option is set both on the CLI and in the inventory, whether the inventory loads from a file or from the `bolt_inventory` environment variable.
-  
+
+* **Support `_run_as` parameter for puppet_library hook** ([#1191](https://github.com/puppetlabs/bolt/issues/1191))
+
+  Bolt now accepts the `_run_as` metaparameter that allows users to specify which user the library install task should be executed as.
```

Comment:
Clarity edit: Bolt now accepts the `_run_as` metaparameter. `_run_as` specifies which user should execute the library install task.
But want to make sure I captured the information correctly.

Reply:
This seems like really it's adding _run_as support to all task-based plugins, including the puppet_library one.